Directory.Build.props

```diff
@@ -6,7 +6,7 @@
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)/jellyfin.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+  <PropertyGroup>
```

Comment:
Now you just remove the `TreatWarningsAsErrors` false from a project if all it's warnings are fixed